src/map/status.cpp

```diff
@@ -11511,6 +11507,19 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 			val2 = (status->int_ + status->luk) * val1 / 20 * status_get_lv(bl) / 200 + val1;	// Chance to evade magic damage.
 			val3 = ((val1 * 15) + (10 * (sd?pc_checkskill(sd,CR_DEFENDER):skill_get_max(CR_DEFENDER)))) * status_get_lv(bl) / 100; // Defence added
 			break;
+		case SC_SHIELDSPELL_HP:
+			val2 = 3; // 3% HP every 3 seconds
```

Comment:
related to Sigma's info its 5% for each 3 seconds
```2.2. Shield Spell
- Level-specific effect will be unified into only 1 effect, changes the skill effect as below.
Level 1 : Recovers 5% of user's MaxHP every 3 seconds for 90 seconds.
Level 2 : Recovers 3% of user's MaxSP every 5 seconds for 90 seconds.
Level 3 : Atk + 150 and Matk + 150 for 90 seconds.```

Reply:
I took that from the kRO skill description. Not sure if Sigma or Gravity made the typo.

Reply:
yeah, im sry depending on kRO info its 5%